### PR TITLE
CI: Install missing packages on opensuse

### PR DESCRIPTION
**Pull request by timwoj**
I ran into these while working on the open-telemetry stuff, and then @ckreibich ran into them separately.

### CODE REVIEW

**Review by ckreibich — APPROVED** [review]
Thanks Tim. :-)

**Comment by timwoj** [comment]
I'll wait for CI to pass before merging.

**Pull request merged**
